src/components/Data/Layout.vue

```diff
@@ -108,19 +106,25 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.sidebar {
-  background: #fff;
-  width: $sidebar-width;
-  position: fixed;
-  top: 0;
-  padding-top: $navbar-height;
-  overflow-y: auto;
+.DataLayout {
   height: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+}
+.DataLayout-sidebarWrapper {
+  flex-basis: $sidebar-width;
+  height: 100%;
+  overflow: auto;
+  box-shadow: 0px 0px 5px 0px rgba(112, 112, 112, 1);
+  z-index: 1;
 }
 
-.content {
-  width: calc(100% - #{$sidebar-width});
-  margin-left: $sidebar-width;
+.DataLayout-contentWrapper {
+  flex-grow: 1;
+  height: 100%;
+  overflow: auto;
   padding: $content-gutter;
+  background-color: #f5f5f5;
```

Comment:
we have a variable with this color already set

Reply:
ok